snap/snapcraft.yaml

```diff
@@ -320,17 +320,20 @@ parts:
     plugin: nil
     after: [godeps]
     override-build: |
+      sudo apt-get -y update
+      sudo apt-get -y install ca-certificates curl gnupg lsb-release
+      curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --batch --yes --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+      echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+      sudo apt-get -y update
+      sudo apt-get -y install docker-ce docker-ce-cli containerd.io
```

Comment:
Absolutely not a blocker because architectures currently supported with CLH are mutex with those, but: This isn't packaged for all architectures, while the in-repo `docker.io` is. Are there specific reasons to use `download.docker.com`?

Reply:
> Are there specific reasons to use download.docker.com?

Just following what's suggested in https://docs.docker.com/engine/install/ubuntu/